app/src/main/res/layout/activity_main.xml

```diff
@@ -8,14 +8,25 @@
     tools:context=".activity.MainActivity">
 
     <TextView
-        android:id="@+id/text_view_main_app_title"
+        android:id="@+id/text_view_main_activity_app_title"
         style="@style/TextStyleAppTitle"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginTop="@dimen/main_activity_title_margin_xsmall"
+        android:layout_marginTop="@dimen/main_activity_margin_xsmall"
         android:text="@string/app_name"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/recycler_view_main_activity"
+        style="@style/RecyclerViewWeatherBackground"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/main_activity_margin_xsmall"
+        android:layout_marginBottom="@dimen/main_activity_margin_xsmall"
```

Comment:
I think that you can add these two margins in the style

Reply:
done